workflows/templates/bandit-template.yaml

```diff
@@ -65,3 +65,61 @@ spec:
           requests:
             cpu: 200m
             memory: 256Mi
+    - name: bandit-from-dir
+      inputs:
+        parameters:
+          - name: "THOTH_SI_BANDIT_JOB_ID"
+          - name: "THOTH_DOCUMENT_ID"
+          - name: "THOTH_SI_BANDIT_PACKAGE_NAME"
+          - name: "THOTH_SI_BANDIT_PACKAGE_VERSION"
+          - name: "THOTH_SI_BANDIT_PACKAGE_INDEX"
+          - name: "THOTH_SI_BANDIT_DIR"
+          - name: "THOTH_S3_ENDPOINT_URL"
+          - name: "THOTH_CEPH_BUCKET_NAME"
+          - name: "THOTH_CEPH_BUCKET_PREFIX"
+          - name: "THOTH_DEPLOYMENT_NAME"
+      outputs:
+        artifacts:
+          - name: outputdocument
+            path: "/mnt/workdir/{{inputs.parameters.THOTH_DOCUMENT_ID}}"
+            archive:
+              none: {}
+            s3:
+              key: "{{inputs.parameters.THOTH_CEPH_BUCKET_PREFIX}}/\
+              {{inputs.parameters.THOTH_DEPLOYMENT_NAME}}/\
+              si-bandit/{{inputs.parameters.THOTH_DOCUMENT_ID}}"
+              endpoint: "{{inputs.parameters.THOTH_S3_ENDPOINT_URL}}"
+              bucket: "{{inputs.parameters.THOTH_CEPH_BUCKET_NAME}}"
+              insecure: true
+              accessKeySecret:
+                name: argo-artifact-repository-secrets
+                key: accessKey
+              secretKeySecret:
+                name: argo-artifact-repository-secrets
+                key: secretKey
+      container:
+        name: si-bandit
+        image: "si-bandit"
+        env:
+          - name: THOTH_DOCUMENT_ID
+            value: "{{inputs.parameters.THOTH_DOCUMENT_ID}}"
+          - name: THOTH_SI_BANDIT_OUTPUT
+            value: "/mnt/workdir/{{inputs.parameters.THOTH_DOCUMENT_ID}}"
+          - name: THOTH_SI_BANDIT_PACKAGE_NAME
+            value: "{{inputs.parameters.THOTH_SI_BANDIT_PACKAGE_NAME}}"
+          - name: THOTH_SI_BANDIT_PACKAGE_VERSION
+            value: "{{inputs.parameters.THOTH_SI_BANDIT_PACKAGE_VERSION}}"
+          - name: THOTH_SI_BANDIT_PACKAGE_INDEX
+            value: "{{inputs.parameters.THOTH_SI_BANDIT_PACKAGE_INDEX}}"
+          - name: THOTH_SI_BANDIT_DIR
+            value: "{{inputs.parameters.THOTH_SI_BANDIT_DIR}}"
```

Comment:
Will this be a persistent volume shared by containers withing the workflow?

Reply:
Yeah, exactly. `/mnt/workdir/project/`

I am actually going to try and see if I can use a default `null` for `THOTH_SI_BANDIT_DIR`

Reply:
Doesn't seem like I can this seems like it'll be the best option.

Reply:
I'm not sure if I follow what you want to accomplish. `THOTH_SI_BANDIT_DIR` will need to be provided to the cli. Otherwise, it will use a temporary directory:

https://github.com/thoth-station/si-bandit/blob/feb3c481903135619f5b1df834c6f06f85f23416/thoth/si_bandit/cli.py#L85-L87

If you do not want to pass `THOTH_SI_BANDIT_DIR` to the workflow in thoth-common, you can remove  `THOTH_SI_BANDIT_DIR` from `inputs.parameters` and hardcode it in this env section to:

```yaml
          - name: THOTH_SI_BANDIT_DIR
            value: "/mnt/workdir/project/"
```

Similarly to `THOTH_SI_BANDIT_OUTPUT` as you've done above. As `/mnt/workdir/` is set up in the workflow (and workflow knows this location) it's probably more clear solution (thoth-common does not need to be aware of the dir location).

Reply:
I still want the option to use it without having `THOTH_SI_BANDIT_DIR` set though hence two separate workflows.

Reply:
> I still want the option to use it without having `THOTH_SI_BANDIT_DIR` set though hence two separate workflows.

Why do you want to have that option?

Reply:
Rn we have two functions in `thoth.common` to schedule them. If we don't have this ability then we need to remove those two functions from `thoth.common`

Reply:
Do we need to maintain the one with `THOTH_SI_BANDIT_DIR`? It feels to me it's a directory used internally inside the workflow and we do not need to have it configurable on workflow level - maybe I'm missing something though.

Reply:
I can remove the functions to schedule si-cloc and si-bandit and let it all be handled by the workflow.